csaf_2.1/prose/edit/src/schema-elements-02-props-01-document.md

```diff
@@ -230,6 +230,14 @@ Document notes (`notes`) of value type Notes Type (`notes_t`) holds notes associ
     },
 ```
 
+The following combinations of `category` and `title` have a special meaning and MUST be used as stated below:
+
+| `category` | `title` | content of `text` |
+|---------------|---------------|-------------------|
+| `description` | Product description | Contains a description of a product given in the `product_tree` in regards to field of application and core functionality. |
```

Comment:
"Title Case" for title cells (Product Description)?

Reply:
done